package errors

import "fmt"

//MainErrors test
func MainErrors() {
	fmt.Println("Start MainErrors >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")
	Error1()
	Error2()
	Panic1()
	fmt.Println("End MainErrors <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<")
	fmt.Println()
}
